tests/test_python/code/README.mkd

```diff
@@ -0,0 +1,12 @@
+Hello World
```

Comment:
Is this one of the unnecessary files you were talking about? I agree: it's unnecessary (unless I'm missing something).